public/js/utils/sources-tree.js

```diff
@@ -5,8 +5,8 @@ const { isPretty } = require("../utils/source");
 const IGNORED_URLS = ["debugger eval code", "XStringBundle"];
 
 function isHiddenSource(source) {
-  const url = getURL(source);
-  return !url || url.path.match(/SOURCE/) || IGNORED_URLS.includes(url);
+  const url = source.get("url");
+  return !url || url.match(/SOURCE/) || IGNORED_URLS.includes(url);
```

Comment:
Oops, you're right, I didn't mean sourcemapped sources. This is for eval sources. But we can't change the ID of eval'ed sources, so this makes sense. Ignore my above comment.